specification/ml/_types/Detector.ts

```diff
@@ -67,15 +71,50 @@ export class Detector {
 }
 
 export class DetectorRead implements OverloadOf<Detector> {
-  by_field_name?: Field
-  custom_rules?: DetectionRule[]
-  detector_description?: string
-  detector_index?: integer
+  /**
+   * The field used to split the data. In particular, this property is used for analyzing the splits with respect to their own history. It is used for finding unusual values in the context of the split.
```

Comment:
I would expect most of these descriptions to match what's in the Detector type.  Is there a reason why we can't inherit the descriptions by default too?